Sources/GATTScanIntervalWindow.swift

```diff
@@ -0,0 +1,63 @@
+//
+//  GATTScanIntervalWindow.swift
+//  Bluetooth
+//
+//  Created by Carlos Duclos on 7/11/18.
+//  Copyright © 2018 PureSwift. All rights reserved.
+//
+
+import Foundation
+
+/**
+ Scan Interval Window
+ 
+ The Scan Interval Window characteristic is used to store the scan parameters of the GATT Client. Included in this characteristic are the Scan Interval and Scan Window of the GATT Client device.
+ 
+ The Scan Interval Window characteristic is used to store the scan parameters of the GATT Client. The GATT Server can use these values to optimize its own advertisement rate and to minimize the rate of its own advertisements while also minimizing the latency of reconnections.
+ 
+ - SeeAlso: [Scan Interval Window](https://www.bluetooth.com/specifications/gatt/viewer?attributeXmlFile=org.bluetooth.characteristic.scan_interval_window.xml)
+ */
+public struct GATTScanIntervalWindow: GATTCharacteristic {
+    
+    internal static let length = MemoryLayout<UInt16>.size * 2
+    
+    public static var uuid: BluetoothUUID { return .scanIntervalWindow }
+    
+    public var scanInterval: UInt16
```

Comment:
What does this represent? We need another struct to represent this value and convert it to milliseconds or seconds.